tools/pythonpkg/tests/fast/test_multithread.py

```diff
@@ -488,5 +503,12 @@ def test_description(self, duckdb_cursor, pandas):
 
     @pytest.mark.parametrize('pandas', [NumpyPandas(), ArrowPandas()])
     def test_cursor(self, duckdb_cursor, pandas):
+        def only_some_succeed(results: List[bool]):
```

Comment:
This could probably be reduced to `lambda results: any(results) and not all(results)`

Reply:
For tests I prefer readability
The old version of this test did not make it clear at all what was happening and being expected

My goal here was to make the expectations much clearer